docs/business-apps/power-automate/guidance/migrate-from-classic-workflows-to-power-automate-flows.md

```diff
@@ -48,6 +48,18 @@ To create and author flows, users primarily use [Power Automate website](https:/
 
 To learn more about building workflows using Power Automate in SharePoint, start here: Business apps and Business process [automation in SharePoint](https://docs.microsoft.com/sharepoint/dev/business-apps/introduction-to-sharepoint-business-process-integration).
 
+## Feature gaps between classic workflows in SharePoint and Power Automate flows
+
+Many people feel there are significant feature gaps between SharePoint Designer (classic) workflows and Power Automate flows, but the list is not long. Of course, there are some gaps and workarounds you should consider in your planning as you move from classic workflows to Power Automate flows.
+	
+* **30 day run limit for flows** – SharePoint Designer workflows can run endlessly, but flows have a 30 day lifespan. Getting beyond this limitation means your flow will need to call itself in a re-entrant way to restart the clock ticking.
```

Comment:
Please keep the markdown bullet format consistent in the doc. Changing the format (from `-` to `*` in this case) triggers MSDocs build warnings & errors.

See the bullets on lines 22-23 for reference.